### PR TITLE
Fix various spelling, grammatical, and punctuation errors

### PR DESCRIPTION
**Pull request by nbriggs**
All space-delimited lowercase and capitalized words were extracted and run through a spelling checker.
Obvious spelling errors were corrected, along with some grammatical and punctuation errors noticed in the course of correcting the spelling.

### CODE REVIEW

**Review by waywardmonkeys — APPROVED** [review]
Wow. LGTM. Really like the "Maxi-MooM" fix.

File: src/mnxmeth.c [hunk 1]
```diff
@@ -169,7 +169,7 @@ int gettitleheight(LispPTR medleywin)
 
 /**************************************************************/
 /* setlineattributes                                          */
-/* This function attempts to syncronize the line data in lisp */
+/* This function attempts to synchronize the line data in lisp */
```
Comment (waywardmonkeys) [inline src/mnxmeth.c:172]: In my local typo branch, I had line-wrapped `lisp` down to the next line and turned it to `Lisp`.

File: src/mnxmeth.c [hunk 2]
```diff
@@ -1497,12 +1497,12 @@ MNXdrawpoint(LispArgs args) /* args[0] = LispPTR to MedleyWindow */
...
 /**************************************************************/
-/* MNXdrawline                                                  */
+/* MNXdrawline                                                */
 /* color not yet implemented                                  */
 /* Medleys Drawline can't handle dashing.                     */
-/* Thick lines in Medley are handled as parallellograms       */
+/* Thick lines in Medley are handled as parallelograms        */
 /* instead of rectangles.                                     */
-/* Remember that this method has an OPERATION arument in Lisp */
+/* Remember that this method has an OPERATION argument in Lisp */
```
Comment (waywardmonkeys) [inline src/mnxmeth.c:1505]: I had line-wrapped `Lisp` down to the next line here too.

File: src/subr.c [hunk 3]
```diff
@@ -299,7 +299,7 @@ void OP_subrcall(int subr_no, int argnum) {
       break;
     case sb_COLOR_BASE:
       POP_SUBR_ARGS;
-      /* retun DLword offsetbetween LISPBASE and Lisp_world */
+      /* return DLword offsetbetween LISPBASE and Lisp_world */
```
Comment (waywardmonkeys) [inline src/subr.c:302]: Missing space in `offsetbetween`

Comment (nbriggs) [reply src/subr.c]: Thanks for catching that.  Must have scanned right over it as I went through the list.

File: src/mnxmeth.c (hunk 1 above)
Comment (nbriggs) [reply src/mnxmeth.c]: I've made that change.

File: src/mnxmeth.c (hunk 2 above)
Comment (nbriggs) [reply src/mnxmeth.c]: Also fixed.

**Pull request merged**